R/synthesis_options_panel.R

```diff
@@ -98,6 +98,13 @@ synthesis_options_panel_ui <- function(id) {
 synthesis_options_panel_server <- function(id, data) {     
   moduleServer(id, function(input, output, session) {
     
+    # Disable Bayesian buttons whilst functionality is not complete #
+    
+    shinyjs::disable(id = "prior")
```

Comment:
If you add `id=ns("bayesian_options"),` to the top-level `div` in this module, then all of the contents, including the help button, can be disabled with one line: `shinyjs::disable(id = "bayesian_options")`